tests/components/plex/test_server.py

```diff
@@ -127,109 +130,81 @@ async def test_new_ignored_users_available(hass, caplog):
+        data=DEFAULT_DATA,
+        options=DEFAULT_OPTIONS,
+        unique_id=DEFAULT_DATA["server_id"],
+    )
 
-#         with patch("plexapi.server.PlexServer", return_value=mock_plex_server), patch(
-#             "homeassistant.components.plex.PlexWebsocket.listen"
-#         ):
-#             entry.add_to_hass(hass)
-#             assert await hass.config_entries.async_setup(entry.entry_id)
-#             await hass.async_block_till_done()
+    mock_plex_server = MockPlexServer()
 
-#         server_id = mock_plex_server.machineIdentifier
+    with patch("plexapi.server.PlexServer", return_value=mock_plex_server), patch(
+        "plexapi.myplex.MyPlexAccount", return_value=MockPlexAccount()
+    ), patch(
+        "homeassistant.components.plex.PlexWebsocket", autospec=True
+    ) as mock_websocket:
+        entry.add_to_hass(hass)
+        assert await hass.config_entries.async_setup(entry.entry_id)
+        await hass.async_block_till_done()
 
-#         with patch.object(mock_plex_server, "clients", return_value=[]), patch.object(
-#             mock_plex_server, "sessions", return_value=[]
-#         ) as mock_update:
-#             # Called immediately
-#             async_dispatcher_send(hass, PLEX_UPDATE_PLATFORMS_SIGNAL.format(server_id))
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 1
+    server_id = mock_plex_server.machineIdentifier
+    loaded_server = hass.data[DOMAIN][SERVERS][server_id]
 
-#             # Throttled
-#             async_dispatcher_send(hass, PLEX_UPDATE_PLATFORMS_SIGNAL.format(server_id))
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 1
+    trigger_plex_update(mock_websocket)
+    await hass.async_block_till_done()
 
-#             # Throttled
-#             async_dispatcher_send(hass, PLEX_UPDATE_PLATFORMS_SIGNAL.format(server_id))
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 1
+    sensor = hass.states.get("sensor.plex_plex_server_1")
+    assert sensor.state == str(len(mock_plex_server.accounts))
 
-#             # Called from scheduler
-#             await self.advance(DEBOUNCE_TIMEOUT)
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 2
+    mock_plex_server.clear_clients()
+    mock_plex_server.clear_sessions()
 
-#             # Throttled
-#             async_dispatcher_send(hass, PLEX_UPDATE_PLATFORMS_SIGNAL.format(server_id))
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 2
+    await loaded_server._async_update_platforms()
+    await hass.async_block_till_done()
 
-#             # Called from scheduler
-#             await self.advance(DEBOUNCE_TIMEOUT)
-#             await hass.async_block_till_done()
-#             assert mock_update.call_count == 3
+    sensor = hass.states.get("sensor.plex_plex_server_1")
+    assert sensor.state == "0"
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=5760157)